artie/manifest.json

```diff
@@ -0,0 +1,68 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ccc5034-0902-4e00-8a69-04d007d31310",
+  "app_id": "artie",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Artie offers real-time replication between databases and data warehouses.",
+    "title": "Artie",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Artie overview dashboard",
+        "image_url": "images/artie-overview-dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Rows processed chart",
+        "image_url": "images/rows-processed-chart.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Replication slot size chart",
+        "image_url": "images/replication-slot-chart.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Cloud",
+      "Category::Data Stores",
+      "Category::Metrics",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "artie overview": "assets/dashboards/artie_overview.json"
+    },
+    "integration": {
+      "auto_install": false,
+      "source_type_name": "Artie",
```

Comment:
This is currently missing a `source_type_id` because I'm not sure what to use for it. I see in the docs that this should have been generated when I ran `ddev create`, but I don't see an integer ID anywhere in the output from that.